v1/lib/output.php

```diff
@@ -47,7 +47,11 @@ public function runGame($_cycle, $_mode)
                 // Make rows
                 for($j=0; $j<$this->x; $j++)
                 {
```

Comment:
Your output class is not needed anymore. Please remove it. (save it locally so that you can reuse the code in your new plugins)